package/koreader/koreader

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-# Copyright (c) 2020 The Toltec Contributors
+# Copyright (c) 2021 The Toltec Contributors
 # SPDX-License-Identifier: MIT
-
-KO_DONT_GRAB_INPUT=1 bash /opt/koreader/koreader.sh
+LD_PRELOAD=/opt/lib/librm2fb_client.so.1 KO_DONT_GRAB_INPUT=1 exec bash /opt/koreader/koreader.sh
```

Comment:
this `exec` is confusing remux in regards to koreader. (I think it replaces the current process and now remux can't find koreader.sh? it doesn't make sense to me yet)

Reply:
Thanks for investigating that! Yes, exec replaces the parent process. It makes sense to use it here because we don’t need the parent Bash process to linger after it has launched KOReader. I think we can use the (Bash-specific) `-a` flag to change the name of the replacing process and thereby avoid confusing remux. I’ll try it out.